up-and-running/customization.md

```diff
@@ -1,31 +1,145 @@
-to place an HTTPs load-balancer in front of npm Enterprise, for example:
+While you may choose to use the default registry URL
+(`https://my-server:8080`), setting an alternative port # will not result
+in the registry binding to an alternative port (it
+  always binds to `:8080`).
 
-[using nginx for SSL termination](https://gist.github.com/bcoe/c073859d060b802b15aa)
+### Full URL of npm Enterprise website
+
+The externally accessible URL of your npm Enterprise website.
+
+### Your company name
+
+The company name that should be displayed on the npm Enterprise website.
+
+### Secret used between services
+
+The secret key used both between internal services and to connect
+upstream replication.
+
+### Storage
+
+Configure the storage for npm Enterprise's various data-files. Frequently you
+will need to switch these paths to match a larger external drive that
+you've mounted on your appliance.
+
+### Scoped CLI Search
+
+By default `npm search` does not return scoped packages (since this is
+  potentially a breach of private information). Enable this setting to
+start returning scoped search results to the CLI.
+
+_Note: you should only do this if your npm Enterprise appliance is
+  running behind a firewall._
+
+### Read Through Cache
+
+By default, rather than mirroring the entire `>1TB` npm registry,
+npm Enterprise selectively caches packages the first time that you
+install them:
+
+```
+npm i lodash --registry=http://my-private-registry:8080
+```
+
+If you would rather selectively whitelist packages from the public registry,
+or would prefer to mirror the entire registry, set this option to `No`.
+
+### Reject Unauthorized
+
+This this option to `No`, if you would like to disable strict SSL checks
+for operations performed by npm Enterprise.
+
+_As an example, you might disable SSL checks if your upstream GitHub enterprise
+server is using a self-signed certificate._
+
+### Upstream Registry
+
+What upstream registry should npm Enterprise replicate packages from? This setting
+defaults to the public registry (`https://replicate.npmjs.com`), but can be
+configured to replicate from other internal npm Enterprise registries for HA
+deployments.
+
+* **Upstream URL**: the registry to replicate from.
+* **Upstream secret**: if connecting to another npm Enterprise registry, provide
+  the _Secret used between services_ of the upstream server.
+* **Policy to apply during replication**: defaults to `white-list`, only replicating
+  modules added to your whitelist file (either by an administrator, or by the
+  read-through-cache). To replicate an entire upstream registry, set this value
+  to `mirror`.
+
+### Auth Scoped Installs
+
+Should authentication be required to install scoped modules?
+
+### Auth All Installs
+
+Should authentication be enabled to install both scoped (`@company/foo`), and
+global modules from your registry (`lodash`).
+
+### Auth Website
+
+Should authentication be required to view the npm Enterprise website.
```

Comment:
This is fine, but could possibly be a little more specific:

> Should a user be required to authenticate (via login form) in order to use your private npm Enterprise website?
>
> This setting defaults to `No` so that package names, descriptions, and READMEs are easily accessible within your organization. If your npm Enterprise instance is not running behind a firewall, you may want to set this to `Yes`.